### ISSUE TITLE
Container initialization takes too long

### ISSUE THREAD

**Issue by laszlocsontos** [craftingjava/docker-suitecrm#5]
Changing the ownership of the contents of `/var/www/html` takes too long when the container boots for the first time.

While this doesn't take a significant time locally, provisioning a pod on GKE (Google Kubernetes Engine) is problematic.

As a workaround `initialDelaySeconds` of [liveness and readiness probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/) are now set to 3 minutes, because that much time is surely enough for all the initialization steps to take place.

**Nevertheless, this could be greatly optimized.**

Instead of changing the ownership in the bootstrap script from [here](https://github.com/craftingjava/docker-suitecrm/blob/master/bootstrap/suitecrm-init.sh#L100), that could be done at image creation time at [this](https://github.com/craftingjava/docker-suitecrm/blob/master/Dockerfile#L53) point. When `suitecrm-init.sh` has finished, then only the permission of those files would need to be changed which were created or altered by the initialization process.

**Comment by kubanczyk** [comment]
As a part of this, I'll migrate crontab from root to www-data (recommended by https://docs.suitecrm.com/blog/scheduler-jobs/ and by common sense).

**Comment by kubanczyk** [comment]
I corrected the runtime issue, although I cannot look at GKE myself for lack of docs/specs.

The main speedup was from running install.php under `su www-data`, as the rest of the php runs. This way no manual chown is needed to correct broken ownership.

As the chowns and chmods turned out to be also the longest parts of `docker build`, I have dug through this to the bottom to also streamline the build.

I re-examined the docs https://docs.suitecrm.com/admin/installation-guide/downloading-installing/

These recommend to chown/chmod right after `tar x`, so I assure it at the same moment. 
To avoid all `chown` and a large part of `chmod`, I'm now doing `RUN tar x` and `RUN php compose` under `USER www-data`. This means we don't break ownership by running as root, so we don't need to correct everything. I've verified with `man tar` that the ownership and permissions would still be set as expected. I don't see any binary files that would need execute bit, so I'm not setting it explicitly for files. I set it for directories.

The `compose` docs don't indicate need to manually correct chmod/chown.

The `chmod 775 cache` is <strike>an error in SuiteCRM docs</strike> unsuitable because we're using undocumented tar.gz download instead of the documented zip download. They are different. The installation procedure as provided would never succeed: "no such directory". I'm omitting `cache` as I discovered it gets created only by install.php, which assigns more sophisitcated permissions and likely our primitive chmod will only last until next cache change.

There is that one remaining `chmod 775` for selected directories that is my opinion redundant. I'm keeping it only because of docs. There is no uid on docker that could possibly use that. Removing that chmod would considerably speed up the build further.
